libp2p/netMessenger_test.go

```diff
@@ -2160,3 +2071,123 @@ func TestNetworkMessenger_BroadcastUsingPrivateKey(t *testing.T) {
 		assert.Equal(t, 0, messages[mes2.ID()])
 	}
 }
+
+func TestNetworkMessenger_AddPeerTopicNotifier(t *testing.T) {
```

Comment:
this test func is exactly the same as on elrond-go:
[proof](https://www.diffchecker.com/Ytmu123O)